internal/groups/groups.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
```

Comment:
Why we didn't put groups in `/pkg` ? Its not a topic of this PR , but I am curious. Looks like a good candidate to me.

Reply:
initialy I've put it there, but then I was asked to move it here, `pkg` holds packages that usually get imported into other  projects I think that was the reasoning.

Reply:
IMHO `/pkg` is the right place for this. It's sharable and exported :)  True, but `internal` is for things that you explicitly don't someone import. Nvm, it's out of the scope of this PR. Thanks for the info